web/src/data/api.ts

```diff
@@ -701,6 +701,18 @@ function initiateOrcidLogin(state: string = '') {
 
 const REFRESH_TOKEN_KEY = 'storage.refreshToken';
 
+function getRefreshToken() {
+  return window.localStorage.getItem(REFRESH_TOKEN_KEY);
+}
+
+function setRefreshToken(token: string) {
+  return window.localStorage.setItem(REFRESH_TOKEN_KEY, token);
+}
+
+function clearRefreshToken() {
+  return window.localStorage.removeItem(REFRESH_TOKEN_KEY);
+}
```

Comment:
I think I had initially thought to put them in `web/src/store/index.ts` but that resulted in circular imports between it and `web/src/data/api.ts`. So I've now moved them to a new module, `web/src/store/localStorage.ts`. I also moved a bit of code from `web/src/store/index.ts` into the new module so that everything that interacts with `window.localStorage` is together there.

Down the road it would probably be good to properly type the get/setQueryState functions, but importing the `Conditions` interface from `web/src/data/api.ts` leads to circular imports again. Rather than _also_ moving a bunch of interface definitions around I just left it this way for now.